crates/compass/src/scraper/source.rs

```diff
@@ -59,8 +61,12 @@ pub(super) struct Jurisdiction {
 pub(super) struct Document {
     /// Source of the document, such as a URL
     source: String,
-    /// Year of the ordinance, such as 2023
-    ord_year: u16,
+    /// Day that the ordinance went into effect, such as 4
+    effective_day: Option<u16>,
+    /// Month that the ordinance went into effect, such as 27
+    effective_month: Option<u16>,
+    /// Year that the ordinance went into effect, such as 2023
+    effective_year: Option<u16>,
```

Comment:
Isn't year required?

Reply:
No, sometimes it's not possible to tell what year the document came from, so it can be null